projects/plugins/wpcomsh/private-site/private-site.php

```diff
@@ -647,7 +647,7 @@ function preprocess_comment( $comment ) {
  * @return bool
  */
 function is_jetpack_connected() {
-	return is_callable( 'Jetpack::is_connection_ready' ) && Jetpack::is_connection_ready();
+	return class_exists( 'Jetpack' ) && Jetpack::is_connection_ready();
```

Comment:
Looks like a reasonable change, but doesn't seem to have anything to do with the PHP 7.2 or WP changes.